staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -44,30 +41,9 @@ const (
 
 // GetTransformerOverrides returns the transformer overrides by reading and parsing the encryption provider configuration file
 func GetTransformerOverrides(filepath string) (map[schema.GroupResource]value.Transformer, error) {
-	f, err := os.Open(filepath)
-	if err != nil {
-		return nil, fmt.Errorf("error opening encryption provider configuration file %q: %v", filepath, err)
-	}
-	defer f.Close()
-
-	result, err := ParseEncryptionConfiguration(f)
-	if err != nil {
-		return nil, fmt.Errorf("error while parsing encryption provider configuration file %q: %v", filepath, err)
-	}
-	return result, nil
-}
-
-// ParseEncryptionConfiguration parses configuration data and returns the transformer overrides
-func ParseEncryptionConfiguration(f io.Reader) (map[schema.GroupResource]value.Transformer, error) {
-	configFileContents, err := ioutil.ReadAll(f)
-	if err != nil {
-		return nil, fmt.Errorf("could not read contents: %v", err)
-	}
-
-	var config EncryptionConfig
-	err = yaml.Unmarshal(configFileContents, &config)
+	config, err := loadConfigFromFile(filepath)
```

Comment:
@liggitt Do we have a requirement to fully support old format of the config file? Perhaps we should show a warning in this case?

Reply:
I assume that no, we don't have such requirement.